stormgls/stormgls.py

```diff
@@ -557,8 +577,20 @@ async def lsinit(ls: StormLanguageServer, params: types.InitializeParams):
     not existing even though it totally does, all because loadCompletions has not
     finished running
     '''
-    async with getTestCore() as core:
-        await ls.loadCompletions(core)
+    cache = (CWD / 'completions.mpk').absolute()
+
+    if not cache.exists() or cache.stat().st_size == 0:
+        completions = await saveCompletions(cache)
+
+    else:
+        # ls.show_message(f'Loading {cache} {cache.stat().st_size}')
+        completions = s_msgpack.un(cache.read_bytes())
+
+        if (version := completions['version']) != s_version.verstring:
```

Comment:
Do you recall if the model hash stuff has landed (and if it requires a live cortex to fetch or if it's baked in somewhere)?

Reply:
Per gobacks: model hashing isn't gonna be an open source thing.